pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -340,6 +365,9 @@ func (config *V2RegistriesConf) postProcessRegistries() error {
 		if err != nil {
 			return err
 		}
+		if registry[0:2] == "*." {
+			registry = strings.Replace(registry, "*.", "", 1)
+		}
```

Comment:
That seems unrelated to the PR.  If we wanted to support a new syntax for unqualified search registries, we'd be running into the same problem as with the `prefix`: breaking backwards compat with older versions.

Reply:
yup, old stuff I forgot to delete. Will be removed.